src/main/java/com/iwaconsolti/league/model/PlayerModel.java

```diff
@@ -4,6 +4,8 @@
 
 @Data
 public class PlayerModel {
-    private int idJugador;
-    private String nombreJugador;
-}
+    private int idpLayer;
```

Comment:
Consider renaming 'idpLayer' to 'idPlayer' to maintain camelCase and readability.
Similar rename suggestion: 'idLeague'
Consider renaming to 'playerName' for clarity.